app/services/discovery_report.rb

```diff
@@ -70,26 +66,19 @@ def registration_check(druid)
   # @return [String] primitive version
   def header
     fields = ['Object Container', 'Number of Items', 'Files with no ext', 'Files with 0 Size', 'Total Size', 'Files Readable']
-    fields.concat ['Label', 'Source ID'] if using_manifest?
+    fields.concat ['Label', 'Source ID']
     fields.concat ['Num Files in CM Manifest', 'All CM files found'] if using_smpl_manifest?
-    fields.concat ['Duplicate Filenames?', 'DRUID', 'Registered?', 'APO exists?']
-    fields << 'SourceID unique in DOR?' if using_manifest?
+    fields.concat ['Duplicate Filenames?', 'DRUID', 'Registered?', 'APO exists?', 'SourceID unique in DOR?']
```

Comment:
whoops - can get rid of "SourceID unique in DOR?" header -- sorry I didn't notice on first pass.

Reply:
Seems like we can get rid of headers completely, if we are making the report output JSON.